lib/expression.js

```diff
@@ -741,18 +781,34 @@ CronExpression.prototype.reset = function reset (newDate) {
   this._currentDate = new CronDate(newDate || this._options.currentDate);
 };
 
+/**
+ * Stringify the expression
+ *
+ * @public
+ * @param {Boolean} [includeSeconds] Should stringify seconds
+ * @return {String}
+ */
+CronExpression.prototype.stringify = function stringify (includeSeconds) {
```

Comment:
```suggestion
CronExpression.prototype.stringify = function stringify(includeSeconds) {
```

Space or no space. I can add linter rule later and clean it up as there some mixed patterns in `master` branch too. I myself prefer no extra space here. But yeah,  rather low priority issue.

Reply:
fixed